src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html

```diff
@@ -0,0 +1,213 @@
+<h1 id="top" class="color-orange">Getting Started</h1>
+<div id="contentHolder">
+  <p>
+    Welcome to TEAMMATES!
+  </p>
+  <p>
+    To get started using TEAMMATES, follow the following steps, or watch our <a href="https://www.youtube.com/embed/mDtfmNmRwBM?autoplay=1&rel=0" target="_blank"><i class="fas fa-film"></i> Video Tour</a>.<br>
+    For more help, browse the answers to some <a routerLink="/web/front/help/instructor">frequently asked questions</a>.
+  </p>
+  <ol>
+    <li><a (click)="jumpTo('course-setup')" [routerLink]="">Set up a course</a></li>
+    <li><a (click)="jumpTo('session-setup')" [routerLink]="">Create a session</a></li>
+    <li><a (click)="jumpTo('session-invites')" [routerLink]="">Wait for your session to open</a></li>
+    <li><a (click)="jumpTo('session-results')" [routerLink]="">View and publish session results</a></li>
+    <li><a (click)="jumpTo('other-actions')" [routerLink]="">Learn about other actions you can perform</a></li>
+    <li><a (click)="jumpTo('contact-us')" [routerLink]="">Contact us</a></li>
+  </ol>
+  <div class="separate-content-holder">
+    <hr>
+  </div>
+  <h2 id="course-setup">1. Set up a course</h2>
+  <div>
+    <p>
+      A course is how TEAMMATES organises feedback sessions. Each course contains instructors, students and sessions specific to the course.
+    </p>
+    <ol>
+      <li>
+        <b>Create a course</b><br>
+        From the <b>Home</b> page, click <button class="btn btn-primary btn-sm">Add New Course</button>.<br>
+        Fill out the following form. Hover your mouse over text to reveal tooltips which tell you what the element does.
+        <p><b>EXAMPLE BOX</b></p>
+      </li>
+      <li>
+        <b><a routerLink="../instructor" fragment="course-add-students" target="_blank" rel="noopener noreferrer">Enroll students in the course</a></b><br>
```

Comment:
If using `routerLink` you don't need `target="_blank" rel="noopener noreferrer"` anymore.

Reply:
Fixed!